lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -3805,8 +3835,13 @@ estimateSignedTxSizeSpec = describe "estimateSignedTxSize" $ do
                 Cardano.TxInsCollateralNone -> []
             ins = Cardano.txIns body
 
-        paymentAddr = Address $ unsafeFromHex
-            "6079467c69a9ac66280174d09d62575ba955748b21dec3b483a9469a65"
+
+
+    bootAddr = Address $ unsafeFromHex
+        "82d818582183581cba970ad36654d8dd8f74274b733452ddeab9a62a397746be3c42ccdda0001a9026da5b"
+
+    vkCredAddr = Address $ unsafeFromHex
+        "6079467c69a9ac66280174d09d62575ba955748b21dec3b483a9469a65"
```

Comment:
Similarly, can we add a comment to describe by what exact process this address was generated? 

Alternatively, is there a way to generate the address without hard-coding a magic hexadecimal value?